tinycrypt/ecc_dsa.c

```diff
@@ -278,9 +287,10 @@ int uECC_verify(const uint8_t *public_key, const uint8_t *message_hash,
 
 	/* Accept only if v == r. */
 	diff = uECC_vli_equal(rx, r);
-	if (diff == 0) {
+	if (diff == 0 && flow_control == (num_bits + 4)) {
```

Comment:
Please add comments about flow control calculation, why it should be num_bits+X

Reply:
Done.